libclc/generic/include/spirv/spirv_types.h

```diff
@@ -46,4 +46,16 @@ enum GroupOperation {
   ExclusiveScan = 2,
 };
 
+typedef struct {
```

Comment:
Would it make sense to add bfloat16 here? It is still 'experimental', but will be moved out soon. We can wait for that to happen. Thanks

Reply:
This has already got rather long in the tooth, so I'd like to defer that change for later